doc/whats_new/v0.24.rst

```diff
@@ -785,6 +785,10 @@ Changelog
 - |Enhancement| :func:`tree.plot_tree` now uses colors from the matplotlib
   configuration settings. :pr:`17187` by `Andreas Müller`_.
 
+- |Fix| :func:`tree.export_graphviz`
```

Comment:
Could you add the entry in `v1.0.rst` (you might need to merge master into your branch first). We already on our way to release 0.24 and we will probably be a bit too late to get this PR in.

Reply:
The `v1.0.rst` file is not in my branch. The `v1.0.rst` was committed to **master** two days ago.

Reply:
Yes you need to merge master into your branch to synchronize with master

Reply:
fine